File: backend/Rest API PWII/Models/ViewModels/CRepostModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Rest_API_PWII.Models.ViewModels
{
    public class CRepostModel
    {
        public string UserID { get; set; }

        public int? PostID { get; set; }

        public DateTime? RepostDate { get; set; }
    }
}

File: frontend/src/model/CRepostModel.js
class CRepostModel{
    userID;
    postID;
    repostDate;

    contructor({userID, postID, repostDate}){
        this.userID = userID;
        this.postID = postID;
        this.repostDate = repostDate;
    }
}

export default CRepostModel;